BetterMD/elements/document.py

```
@@ -0,0 +1,84 @@
+
+class Copy:
+    def __init__(self, data):
+        self.data = data
+    
+    def copy(self):
+        return self.data
+
+T5 = t.TypeVar("T5", bound=CopyProtocol)
+
+class Fetcher(t.Generic[T1, T2, T5]):
+    def __init__(self, data: 'GetProtocol[T1, T2]', default:'T5'=Copy(None)):
```

Ruff (0.8.2)

Do not perform function call `Copy` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)